delft3dworker/models.py

```diff
@@ -498,42 +498,53 @@ def update_and_phase_shift(self):
 
         if self.phase == self.phases.new:
 
-            preprocess_container = Container.objects.create(
-                scene=self,
-                container_type='preprocess',
-                desired_state='non-existent',
-            )
-            preprocess_container.save()
-            delft3d_container = Container.objects.create(
-                scene=self,
-                container_type='delft3d',
-                desired_state='non-existent',
-            )
-            delft3d_container.save()
-            process_container = Container.objects.create(
-                scene=self,
-                container_type='process',
-                desired_state='non-existent',
-            )
-            process_container.save()
-            export_container = Container.objects.create(
-                scene=self,
-                container_type='export',
-                desired_state='non-existent',
-            )
-            export_container.save()
-            postprocess_container = Container.objects.create(
-                scene=self,
-                container_type='postprocess',
-                desired_state='non-existent',
-            )
-            postprocess_container.save()
-            sync_clean_container = Container.objects.create(
-                scene=self,
-                container_type='sync_cleanup',
-                desired_state='non-existent',
-            )
-            sync_clean_container.save()
+            if not self.container_set.filter(container_type='preprocess').exists():
```

Comment:
Assure only one container is created, even if a Scene is put in phases.new again.